test/css_mirroring/transformer_test.dart

```diff
@@ -0,0 +1,231 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+library scissors.test.css_mirroring.transformer_test;
+
+import 'package:barback/barback.dart'
+    show BarbackMode, BarbackSettings, Transformer;
+import 'package:scissors/src/css_mirroring/transformer.dart';
+
+import 'package:code_transformers/tests.dart' show testPhases;
+
+makePhases(Map config) => [
+      [
+        new CssMirroringTransformer.asPlugin(
+            new BarbackSettings(config, BarbackMode.RELEASE))
+      ]
+    ];
+
+void main() {
+  var phases = makePhases({});
+
+  testPhases(
+      'splits the rule with language dependent declaration to common rule and language dependent rules',
+      phases, {
+    'a|foo2_unmatched_css_url.css': r'''
+          absent-element {
+            color: blue;
+            float: right;
+        }
+       '''
+  }, {
+    'a|foo2_unmatched_css_url.css': r'''
+          absent-element {
+            color: blue;
+            }
```

Comment:
i this because the float:right was removed so it aligns the closing bracket to its start.